### PR TITLE
Issue #13533 - Added support to parse instrument recovered CTDBP/FLORTD combined data.

### PR DESCRIPTION
**Pull request by renegelinas**

### CODE REVIEW

File: mi/dataset/parser/flort_dj.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+# Next 6 chars: cccccc = Conductivity A/D counts
+# Next 6 chars: pppppp = pressure A/D counts
+# Next 4 chars: vvvv = temperature compensation A/D counts
+# Next 4 chars: bbbb = Backscatter in counts (FLORT data omitted from output)
+# Next 4 chars: llll = Chlorophyll in counts (FLORT data omitted from output)
+# Next 4 chars: dddd = CDOM in counts (FLORT data omitted from output)
+# Last 8 chars: ssssssss = seconds since January 1, 2000
+# Total of 42 hex characters and line terminator
+
+CTD_FLORT_DATA_REGEX = r'(?:' + ASCII_HEX_CHAR_REGEX + '{6})'
```
Comment (danmergens) [inline mi/dataset/parser/flort_dj.py]: why is this duplicated instead of just importing from `mi/dataset/parser/ctdbp_cdef.py`?

Comment (renegelinas) [reply mi/dataset/parser/flort_dj.py]: The regex is different between the two files as far as which data are omitted.  However,  the comment was incorrect in flort_dj.py; copy/paste error.  I've updated the comment to reflect the regex.

**Comment by renegelinas** [comment]
The RELEASE_NOTES have been updated.

**Comment by danmergens** [comment]
Once the travis build completes, you can merge this. Then you'll need to update the RELEASE_NOTES on the other pull request.

**Comment by renegelinas** [comment]
Good, cause that's what I was going to do! 👍

**Pull request merged**